### PR TITLE
made entry for saturday and friday

### PR DESCRIPTION
**Pull request by dgwyn**
please check for errors.

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -1,6 +1,20 @@
 Hi! My name is David, and I am going to tell you what I have learned from my
 dad every day.
 
+## Saturday, May 11
+
+Today my dad told me  to down load a app called Jira. At first I was thinking,
+why would he tell me to down load a app? Then he told me that this is how I can
+get tickets to do things like projects. First he make tickets to do simple
```
Comment (defn) [inline index.md]: Fix `First he make tickets`

File: index.md [hunk 2]
```diff
@@ -1,6 +1,20 @@
 Hi! My name is David, and I am going to tell you what I have learned from my
 dad every day.
 
+## Saturday, May 11
+
+Today my dad told me  to down load a app called Jira. At first I was thinking,
```
Comment (defn) [inline index.md]: Two spaces between words, find these in vim with this command: `/ . ` then look for more occurences with this command: `n`

File: index.md [hunk 3]
```diff
@@ -1,6 +1,20 @@
...
+why would he tell me to down load a app? Then he told me that this is how I can
+get tickets to do things like projects. First he make tickets to do simple
+things, like get him a coffee. Then he assigned to more harder like to make a
```
Comment (defn) [inline index.md]: Fix `he assgined to more harder like...`

File: index.md [hunk 4]
```diff
@@ -1,6 +1,20 @@
...
+PR about Jira. This is the PR I am doing now!
+
+## Friday, May 10
+
+like on tusday, the science project is making me very busy, so I couldn't make
```
Comment (defn) [inline index.md]: Capitalization

File: index.md [hunk 5]
```diff
@@ -1,6 +1,20 @@
...
+time for PRs. Here are the things I did while I was not makeing PRs. I made
```
Comment (defn) [inline index.md]: Mispelling: `makeing`

File: index.md [hunk 6]
```diff
@@ -1,6 +1,20 @@
...
+the rest of my blades, and I did all my testing. So right now, I feel pretty
+good.
```
Comment (defn) [inline index.md:17]: Need whitespace between paragraph and header.

**Review by ghost — CHANGES_REQUESTED** [review]

File: index.md (hunk 1 above)
Comment (dgwyn) [reply index.md]: done.

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**